src/bootstrap/bin/rustc.rs

```diff
@@ -107,6 +119,11 @@ fn main() {
              env::join_paths(&dylib_path).unwrap());
     let mut maybe_crate = None;
 
+    // Print backtrace in case of ICE
+    if env::var("RUST_BACKTRACE").is_err() {
+        cmd.env("RUST_BACKTRACE", "1");
+    }
```

Comment:
Is `RUST_BACKTRACE=1` fast enough to be turned on by default on all hosts? #42295

Reply:
You can turn it off with `RUST_BACKTRACE=0`, if you're hitting ICEs and don't want to look at the backtrace for some odd reason. #37477 should no longer apply since rustc doesn't use panics during normal operation.

Reply:
This wrapper is not used for running tests, right? Then it should be fine.

Reply:
It is not used for running tests. I don't think it's used for compiling tests either.

Reply:
Hm we've had bad enough luck with this in the past that I'd personally prefer to leave this off by default and it can always be opted into by setting it locally, right?

Reply:
I can't think of any scenario where you wouldn't want a backtrace here. I want backtraces on CI so it should be enabled there. Opting into it locally is annoying, and it's easy to forget to turn it off when you want to run tests. What kind of bad luck are you referring to?

Reply:
I could probably make this a config.toml option.

Reply:
Yeah having a config option seems reasonable